MultiServer.py

```diff
@@ -634,13 +634,29 @@ def get_hint_cost(self, slot):
             return max(1, int(self.hint_cost * 0.01 * len(self.locations[slot])))
         return 0
 
-    def recheck_hints(self, team: typing.Optional[int] = None, slot: typing.Optional[int] = None):
+    def recheck_hints(self, team: typing.Optional[int] = None, slot: typing.Optional[int] = None,
+                      changed: typing.Set[tuple[int,int]] = None, already_updated: typing.Set[tuple[int,int]] = None):
```

Comment:
*sigh* fair. Where would I put it, what should I call it?

Reply:
I personally like doing one class per file, but I've heard sentiment against that in core, so maybe right before or right after this one?
I'd call it "TeamPlayer" or "TeamSlot", in that order because "PlayerTeam" sounds just like a player's team, not a player an a team

Reply:
...It was... already declared there. Welp. That makes things easy XD